vrs/marten-construction-script.py

```diff
@@ -0,0 +1,168 @@
+# count_1mil = ht_1mil.count()
+# indel_freq = ht_1mil.filter(hl.is_indel(ht_1mil.alleles[0], ht_1mil.alleles[1])).count() / count_1mil # = 0.14531311994611185
+# ref_freq = ht_1mil.filter(hl.len(ht_1mil.alleles[0])>5).count() / count_1mil # = 0.02129941204747994
+# var_freq = ht_1mil.filter(hl.len(ht_1mil.alleles[1])>5).count() / count_1mil # = 0.03805950683146261
+# x_freq = ht_1mil.filter(ht_1mil.locus.contig == "chrX").count() / count_1mil # = 0.04005632671091385
+# y_freq = ht_1mil.filter(ht_1mil.locus.contig == "chrY").count() / count_1mil # = 0.0015570797658583776
+# print(f'indel freq as: {indel_freq} \n\tlong ref and var freq as: {ref_freq} , {var_freq} \n\tx and y as: {x_freq} , {y_freq}')
+
+indel_freq = 0.1453
+ref_freq = 0.02130 
+var_freq = 0.03806 
+x_freq = 0.04006 
+y_freq = 0.001557 
+whole_count = 759_000_000
+
+# e.g.: for 1000 variants, 146 will be indels
+# for 1000 variants, 40.06 will be X chromosomes and only 1.557 will be Y chromosome
+
+# Construct tables of samples from this 
+# Makeup: 50k random, 10k extra indels, 10k more long references, 10k long variants, 10k sex chromosome
+# --> (9k X, 1k Y)
+
+# 50,000 random samples
+randSampleFreq = 50000/whole_count
+ht_50k = ht_whole.sample(randSampleFreq).repartition(100)
+# TO ADD: ht_50k = ht_50k.repartition(100)
+rand_path = 'gs://gnomad-tmp-4day/marten-temp-01-06/ht_50k_temp.ht'
+ht_50k = ht_50k.checkpoint(rand_path,overwrite = True)
+
+# 10,000 indels
+ht_10k_indel = ht_whole.filter(hl.is_indel(ht_whole.alleles[0], ht_whole.alleles[1]))
+# ht_whole.filter(ht_whole.vep.variant_class != "SNV")
+indel_sample_freq = 10000/(whole_count*indel_freq) # due to repeat samples, looking for 1.5x what we were expecting
+ht_10k_indel = ht_10k_indel.sample(indel_sample_freq).repartition(100)
+# TO ADD: ht_10k_indel = ht_10k_indel.repartition(100)
+indel_path = 'gs://gnomad-tmp-4day/marten-temp-01-06/ht_10k_indel_temp.ht'
+ht_10k_indel = ht_10k_indel.checkpoint(indel_path,overwrite = True)
+
+# 10,000 variants with long reference alleles
+ht_10k_longRef = ht_whole.filter(hl.len(ht_whole.alleles[0])>5)
+longRef_sample_freq = 10000/(whole_count*ref_freq)
+ht_10k_longRef = ht_10k_longRef.sample(longRef_sample_freq).repartition(10)
+# TO ADD: ht_10k_longRef = ht_10k_longRef.repartition(100)
+longRef_path = 'gs://gnomad-tmp-4day/marten-temp-01-06/ht_10k_longRef_temp.ht'
+ht_10k_longRef = ht_10k_longRef.checkpoint(longRef_path,overwrite = True)
+
+# 10,000 variants with long variant alleles
+ht_10k_longVar = ht_whole.filter(hl.len(ht_whole.alleles[1])>5)
+longVar_sample_freq = 10000/(whole_count*var_freq)
+ht_10k_longVar = ht_10k_longVar.sample(longVar_sample_freq).repartition(100)
+# TO ADD: ht_10k_longVar = ht_10k_longVar.repartition(100)
+longVar_path = 'gs://gnomad-tmp-4day/marten-temp-01-06/ht_10k_longVar_temp.ht'
+ht_10k_longVar = ht_10k_longVar.checkpoint(longVar_path,overwrite = True)
```

Comment:
```suggestion
ht_10k_longVar = ht_10k_longVar.checkpoint(longVar_path, overwrite=True)
```

Reply:
(I'll stop commenting on these here and instead add a reminder to run black to reformat this script)